int32 Arm_state
---
int32 state
